docs/configuration/plugins/configuring-g-suite.md

```diff
@@ -48,17 +73,27 @@ for these plugins and the permissions required to enable them. These plugins are
 
 > Used for development to funnel all emails to a specific user.
 
-## G Suite Setup
+### `DWD_ENABLED` \[Optional. Default: True\]
+
+> This variable flags to dispatch that domain-wide delegation is enabled on the service account. This will be a boolean value.
 
-To set up G Suite integration, you'll need to create some resources in Google Cloud Platform, and then link them to your
-G Suite organization.
+### `GOOGLE_CUSTOMER_ID` \[Optional. Default: None\]
+
+> Unique string assigned to each google workspace customer. It can be found by using these [instructions](https://support.google.com/a/answer/10070793?hl=en).
+
+It is needed to use the [Groups API](https://cloud.google.com/identity/docs/how-to/create-google-groups#creating_a_google_group) which doesn't require admin SDK. It is passed as the parent resource under which groups are created, under which memberships are created.
+NOTE: You've to add a 'C' as a prefix to the ID you get from the instructions above.
+
+## Google Workspace Setup
+
+To set up Goolge Workspace integration, you'll need to create some resources in Google Cloud Platform, and then link them to your Google Workspace organization.
```

Comment:
```suggestion
To set up Google Workspace integration, you'll need to create some resources in Google Cloud Platform, and then link them to your Google Workspace organization.
```

Reply:
@kevgliss That is correct. This PR contains the code of that PR which I used as seed. I didn't want to make changes on the same PR so I made a totally different PR. Sorry for the confusion.

Reply:
No worries, I actually prefer a separate PR. I will close out the other one.